gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/base.py.j2

```diff
@@ -99,13 +99,13 @@ class {{ service.name }}Transport(abc.ABC):
         scopes_kwargs = self._get_scopes_kwargs(self._host, scopes)
 
         # Save the scopes.
-        self._scopes = scopes_kwargs["scopes"]
+        self._scopes = scopes or self.AUTH_SCOPES
```

Comment:
If I'm understanding this correctly, `scopes_kwargs["scopes"]` was essentially `scopes` but with a check for if the `endpoint` was supplied.  And I am interpreting `scopes or self.AUTH_SCOPES` to mean "user supplied scopes _or_ the default scopes"

Is that accurate?

Reply:
Yep! Since the `endpoint` restriction has been lifted I went back to "user supplied scopes or the default scopes"